client/dashboard/profile-wizard/steps/theme/index.js

```diff
@@ -38,6 +39,7 @@ class Theme extends Component {
 			activeTab: 'all',
 			chosen: null,
 			demo: null,
+			cart: false,
```

Comment:
It took me a second to look through and find what this property was used for.  Maybe we could rename to `isRedirecting` or `isRedirectingToCart` for slightly faster grokking.